### PR TITLE
fix some manual fixes

### PR DESCRIPTION
**Pull request by joooooooooooooooooooooooooooooooooooosh**
Remove a legacy manual fix for ENGG3060 and fix a manual fix for ELEC3115.

### CODE REVIEW

**Review by Joelrwest — APPROVED** [review]
Still so sad about the ELEC3115 stuff :(

File: backend/data/processors/manual_fixes/ENGGFixes.py [hunk 1]
```diff
@@ -88,16 +86,6 @@ def ENGG_2997():
     return "72UOC && (3778 || 3061)"
 
 
-def ENGG_3060():
-    """
-    "original": "Pre-requisite: Completion of 66 UOC from an engineering degree<br/><br/>"
```
Comment (Joelrwest) [inline backend/data/processors/manual_fixes/ENGGFixes.py:93]: Maybe we should at some point look at this years prerequisites vs last years and flag if anything's changed. Only relevant for manual fixes I'd imagine

Comment (Joelrwest) [reply backend/data/processors/manual_fixes/ENGGFixes.py:93]: I guess when whoever did this in 2021 the handbook said "Pre-requisite: Completion of 66 UOC from an engineering degree" and the 2022 now says "Pre-requisite: Completion of 66 UOC"

Comment (joooooooooooooooooooooooooooooooooooosh) [reply backend/data/processors/manual_fixes/ENGGFixes.py:93]: CIRCLES-297 is a card for this, but it hasn't been picked up yet

**Pull request merged**